mixin updates

shenya1x opened the pull request:
mixin updates

kewenchao committed:
Create keke

Tracked-On: OAM-67232
Signed-off-by: kewenchx wenchao.ke@intel.com

sysctstage reviewed (commented):
Autobuild started from pull-request-changes on this PR.

Cactus automatically verified this PR.

For more information, see: /absp-staging/builders/opensource-autobuild/builds/513

kewenchao committed:
mixin updates

Create keke

Change-Id: Ice79e78d495df135112ae151513be795b214d1e9
Tracked-On: OAM-67232
Mixin-Reviewed-On: https://github.com/sysctstage/device-androidia-mixins/pull/58
Signed-off-by: kewenchx wenchao.ke@intel.com
